.hintrc

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please remove the hintrc, i will do a prettier in the future but for now this doesn't belong to you issue.